### DISCUSSION TITLE
Difference between Context on `tests` and on `proto.context`

### DISCUSSION THREAD

**Discussion by Kludex** [long2ice/asynch#77] [Q&A]
Hi @long2ice 👋 

On the test suite, it looks like a custom Context is being created:

https://github.com/long2ice/asynch/blob/7a9c3dcd468b981d05c26e2a631efc933e535837/tests/test_proto/columns/test_intcolumn.py#L14-L17

But then, on `proto/context.py`, I see this `Context`:

https://github.com/long2ice/asynch/blob/7a9c3dcd468b981d05c26e2a631efc933e535837/asynch/proto/context.py#L10

And on `_prepare` I see this: https://github.com/long2ice/asynch/blob/7a9c3dcd468b981d05c26e2a631efc933e535837/asynch/cursors.py#L221

My question is: is the type of `context` on `_prepare` supposed to be the `proto.context.Context`? 🤔

If so, should we check the `context.settings.execution_options` instead? I'm a bit confused about it...

By the way, we are using `asynch`, thanks for it! :)

**Comment by stankudrow** [comment]
@long2ice , I cannot answer this question, could you possibly help me and @Kludex .
